src/components/SimplePagination/_simple-pagination.scss

```diff
@@ -0,0 +1,28 @@
+@import '~carbon-components/scss/globals/scss/vars';
+
+div.simple-pagination-container {
+  display: flex;
+  height: 3rem;
+  justify-content: flex-end;
+  width: 100%;
+  border: 1px solid $ui-03;
+  background-color: $ui-background;
```

Comment:
I believe this needs to be switched to `$ui-01`

Reply:
> Looking at the storybook I still see the gray background.
> 
> ![image](https://user-images.githubusercontent.com/1590966/73384847-16045180-429a-11ea-865e-743e5493b5f8.png)

I noticed Carbon uses gray for their pagination as well. I'm thinking it may have switched in v10?

Reply:
http://react.carbondesignsystem.com/?path=/story/pagination--pagination

Reply:
yes, our gray10 theme though has white color for these elements that are on a gray background

Reply:
that being said. The theme should be handling the change in color..🤔

Reply:
Swapped to `$ui-01`